places/Dublin.md

```diff
@@ -0,0 +1,9 @@
+## Dublin, Ireland
+
+### Places to eat
+- Acapulco
+- Market Bar
+
+### Things to do
+- See the sights
```

Comment:
I find the inside of any roofed buildings that serve alcohol the best sights to behold